native/ios/Podfile.lock

```diff
@@ -34,33 +57,72 @@ PODS:
     - GCDWebServer/Core (= 3.5.4)
   - GCDWebServer/Core (3.5.4)
   - glog (0.3.5)
+  - GoogleAppMeasurement (10.24.0):
```

Comment:
if i run `pod install` i don't get these resolved pods here. My podfile.lock differs a lot. Even i ran `yarn install` before. But i'm using newer cocoapods version and